qgis_deployment_toolbelt/constants.py

```diff
@@ -38,6 +38,36 @@
 # ##################################
 
 
+def get_qdt_logs_folder() -> Path:
+    """Get QDT logs folder. First, it tries to get path from environment variable
+        `QDT_LOGS_DIR`, then it uses the default path: a `logs` subfolder under the QDT
+        working folder.
+
+    Returns:
+        Path: path to the QDT logs folder.
+    """
+    if isinstance(getenv("QDT_LOGS_DIR"), str) and check_path(
+        input_path=Path(expandvars(expanduser(getenv("QDT_LOGS_DIR")))),
+        must_be_a_file=False,
+        must_be_a_folder=True,
+        must_be_writable=True,
+        raise_error=False,
+    ):
+        qdt_logs_folder = Path(expandvars(expanduser(getenv("QDT_LOGS_DIR"))))
+        logger.debug(
+            f"Logs folder set from QDT_LOGS_DIR environment variable: {qdt_logs_folder}"
+        )
+
+    else:
+        qdt_logs_folder = get_qdt_working_directory().joinpath("logs")
+        logger.debug(
+            "Logs folder specified in QDT_LOGS_DIR environment variable "
+            f"{getenv('QDT_LOGS_DIR')} can't be used (see logs above). Fallback on "
```

Comment:
In case `QDT_LOGS_DIR` is not set we have a debug log for invalid value but value was not set. Maybe we should change the checks.

Maybe use `qdt_logs_folder = get_qdt_working_directory().joinpath("logs") `as a default value at the beginning of the function.

Then check if  `QDT_LOGS_DIR` is defined as environnement value and add log if `check_path` is failing.

Reply:
You're right. It's clearly clearer!